game/src/game/player/player_behaviour.rs

```diff
@@ -1329,6 +1366,15 @@ fn player_attack(
     }
 }
 
+/// Restores the player movement velocity after attacking.
+fn player_restore_velocity(
+    mut query: Query<&mut PlayerStats, (With<Player>, Added<CanAttack>)>,
+) {
+    for mut stats in query.iter_mut() {
+        stats.reset_stat(StatType::MoveVelMax);
+    }
```

Comment:
...and then how I reset the velocity stat here. Check `[reset_stat](https://github.com/TheSeekerGame/TheSeeker/pull/159/files#diff-5e83eac0ac88ef244b29c0e643b3d69cb9c99d39965e6eab34e8c86d471d3328R1128)` out too just in case. @odecay